docs/sphinx/user_guide/using_camp.rst

```diff
@@ -0,0 +1,132 @@
+.. _using_camp-label: 
+
+**********
+Using Camp
+**********
+
+Since Camp is a metaprogramming library that is primarily header-only, it is hard
+to find isolated code examples that show Camp's capability. Instead, this page
+shows several real application examples of how Camp is being used currently.
+
+Camp Used in Umpire
+===================
+
+Umpire is a resource management library that allows the discovery, provision, and management of memory on machines 
+with multiple memory devices like NUMA and GPUs. Umpire is also part of the RAJA Portability Suite.
+Umpire's Operations provide an abstract interface to modifying and moving data between Umpire allocators.
```

Comment:
```suggestion
Umpire's Operations provide an abstract interface to modifying and moving data between Umpire Allocators.
```

Operations and Allocators are key concepts in Umpire. Either capitalize both or neither for consistency.